proto/v1/database_service.proto

```diff
@@ -244,6 +251,18 @@ message GetDatabaseSchemaRequest {
   bool sdl_format = 2;
 }
 
+message GetDatabaseSchemaDiffRequest {
+  // The name of the database to retrieve schema.
+  // Format: instances/{instance}/databases/{database}/schema
+  string name = 1 [(google.api.field_behavior) = REQUIRED];
+
+  string schema = 2;
```

Comment:
Maybe this needs a oneof
1) schema like this;
2) change history instances/*/databases/*/changeHistories/*
3) something else later.
?

I'm trying to reduce the string payload in transit.